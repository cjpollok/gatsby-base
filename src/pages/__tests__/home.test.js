import React from 'react';
import { mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

import HomePage, { HomePageBase } from '../home';

describe('HomePage Component', () => {
  it('renders correctly', () => {
    const wrapper = mount(
      <MemoryRouter>
        <HomePage />
      </MemoryRouter>
    );
    const base = wrapper.find(HomePageBase);
    expect(base.render()).toMatchSnapshot();
  });
});
