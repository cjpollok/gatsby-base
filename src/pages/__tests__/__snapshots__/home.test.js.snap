// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component renders correctly 1`] = `
<a
  href="/page-2/"
>
  Go to page 2
</a>
`;
